pkg/cmd/auth/login/login_test.go

```diff
@@ -432,13 +438,21 @@ func Test_loginRun_Survey(t *testing.T) {
 			opts: &LoginOptions{
 				Interactive: true,
 			},
-			askStubs: func(as *prompt.AskStubber) {
-				as.StubPrompt("What account do you want to log into?").AnswerWith("GitHub Enterprise Server")
-				as.StubPrompt("GHE hostname:").AnswerWith("brad.vickers")
-				as.StubPrompt("What is your preferred protocol for Git operations?").AnswerWith("HTTPS")
-				as.StubPrompt("Authenticate Git with your GitHub credentials?").AnswerWith(false)
-				as.StubPrompt("How would you like to authenticate GitHub CLI?").AnswerWith("Paste an authentication token")
-				as.StubPrompt("Paste your authentication token:").AnswerWith("def456")
+			prompterStubs: func(pm *prompter.PrompterMock) {
+				pm.SelectFunc = func(prompt, _ string, _ []string) (int, error) {
+					switch prompt {
+					case "What account do you want to log into?":
+						return 1, nil
```

Comment:
When someone reads this stub, will they understand what is the meaning of constants like "1", since the test does not verify the options presented?

Reply:
Going to experiment with something like:

```go
				pm.SelectFunc = func(prompt, _ string, opts []string) (int, error) {
					switch prompt {
					case "What account do you want to log into?":
						return test.IndexFor(opts, "github.com"), nil
```